docs/README.md

```diff
@@ -26,7 +26,7 @@ source .venv/bin/activate
 python -m pip install -r requirements.txt
 python -m pip install -r ../pytorch-cpu-requirements.txt
 
-# Install iree-turbine itself.
+# Install wave itself.
 # Editable so you can make local changes and preview them easily.
 python -m pip install -e ..
```

Comment:
same as earlier: parent dir vs current dir.

Reply:
This is for editing from the docs directory. That's why the parent directory. I will update the top level README after we land the setup.py changes.